tools/check_format.py

```diff
@@ -78,6 +78,118 @@
 LIBCXX_REPLACEMENTS = {
     "absl::make_unique<": "std::make_unique<",
 }
+
+UNOWNED_EXTENSIONS = {
```

Comment:
Also if there's any of these we want to flag with owners while we're in here, that'd be great.

For most of these I can think of one maintainer owner but not a backup.  I could also relax our checks and allow a single owner?